rime_settings.c

```diff
@@ -6,11 +6,11 @@
 extern RimeApi *rime_api;
 
 static struct ColorSchemeDefinition preset_color_schemes[] = {
+  { NULL, 0, 0 },
   { "aqua", 0xffffff, 0x0a3dfa },
   { "azure", 0xffffff, 0x0a3dea },
   { "ink", 0xffffff, 0x000000 },
-  { "luna", 0x000000, 0xffff7f },
-  { NULL, 0, 0 }
```

Comment:
The NULL `color_scheme_id` marks the end of the array. This change does not work.

Reply:
Sorry, I didn't read the codes carefully.